### DISCUSSION TITLE
Macowins TPI

### DISCUSSION THREAD

**Discussion by guidoenr** [dds-jv/foro#176] [Consultas generales] [answered]
Que tal! queria ver si entendi bien en claro como es la tarea del TPI de macowins. 
Simplemente lo que tenemos que hacer es darle una estructura maven al proyecto , que tenga todas las funcionalidades que estan en el enunciado + test?

O mas adelante va a existir un enunciado que explique mejor? Gracias

**Comment by gastonprieto** [comment, marked as answer]
Buenas @guidoenr 

Este fin de semana les vamos a pasar el link a una tarea de github classroom para hacer el TP, que ya va a estar estructurado el proyecto con Maven y con las herramientas que vimos hoy del coverage y verificación del código. Tendrían que codificar los tests y las clases necesarias para que ande.

Igual esta explicado bien en el classroom

**Reply by gastonprieto** [reply]
@guidoenr  te sirvio la respuesta? si esta ok, fijate de marcarlo como respondido con la tilde.

**Reply by guidoenr** [reply]
Si gaston, perdon.. muchas gracias
